src/test/java/org/folio/edgecommonspring/security/SecurityManagerServiceTest.java

```diff
@@ -25,53 +30,139 @@ class SecurityManagerServiceTest {
 
   private static final String MOCK_TOKEN = "eyJhbGciOiJIUzI1NiJ9eyJzdWIiOiJ0ZXN0X2FkbWluIiwidXNlcl9pZCI6ImQyNjUwOGJlLTJmMGItNTUyMC1iZTNkLWQwYjRkOWNkNmY2ZSIsImlhdCI6MTYxNjQ4NDc5NCwidGVuYW50IjoidGVzdCJ9VRYeA0s1O14hAXoTG34EAl80";
   private static final String LOGIN_RESPONSE_BODY = "{\r\n    \"username\": \"diku_admin\",\r\n    \"password\": \"admin\"\r\n}";
-  private static final String API_KEY = "eyJzIjoiQlBhb2ZORm5jSzY0NzdEdWJ4RGgiLCJ0IjoidGVzdCIsInUiOiJ0ZXN0X2FkbWluIn0=";
+  private static final String API_KEY = "eyJzIjoidGVzdF9hZG1pbiIsInQiOiJ0ZXN0IiwidSI6InRlc3QifQ==";
   private static final String WRONG_KEY = "eyJzIjoiQlBhb2ZORm5jSzY0NzdEdWJ4RGgiLCJ0IjoidGVzdCIsInUiOiJ3cm9uZ191c2VyIn0=";
```

Comment:
See my comment above about the `MOCK_TOKEN` value. I would like to make the same argument here. Is there anything preventing us from using just a dummy string like "mock_token" or similar?

Reply:
the **_api_key_** and **_wrong_key_** having the value created through utility class (contains salt, user & tenant value if decrypted), which is being used in testing compare agains value in ephemerial.properties file.
So, changed value of MOCK_TOKEN only.